client/src/components/InputFields/InputAssignedBy.js

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
```

Comment:
You do not use React but in the other components you do. It might be better do to it consistent.
Furthermore I am not sure, if you write the names in Pascal Case when you do not use React.